motioner.py

```diff
@@ -37,14 +40,17 @@ def get_tex():
     else:
         i_or_we = 'D-rektoratet'
 
+    items = map(convert_newlines, request.form.getlist('items[]'))
+    background = convert_newlines(request.form['background'])
```

Comment:
I was once told that the Python way of doing things is using list comprehensions instead of `map`.
So you could do `items = [convert_newlines(item) for item in reguest.form.getlist('items[]')]

Reply:
I personally kinda prefer `map` just due to it being more functional 😛, but yeah list comprehension wound probably be considered more 'pythonic'. It arguably is better since it keeps `items` as list rather than a generator